src/components/nav/Banner.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from 'next/image';
+import ArrowRightIcon from '../../images/icons/arrow-right.svg';
+
+export function Banner() {
+  const modularspringUrl = process.env.NEXT_PUBLIC_API_URL;
+  return (
+      <div className="bg-[#D9D9D9] py-2 flex gap-8 sm:gap-10 items-center justify-center">
+        <div className="flex items-center">
+          <a
+            href={modularspringUrl}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <p className="text-black font-semibold text-l flex gap-2 hover:underline">
+                [Modularium] is live <Image className="ml-2 brightness-0" src={ArrowRightIcon} width={24} height={24} alt="" />
+            </p>
+          </a>
+        </div>
+      </div>
+  );
+}
```

Comment:
This should only render if `modularspringUrl` has a value.  We should also make this more generic imho and have both the url and text be configurable via env vars.